gatsby-theme-landing-page/README.md

```diff
@@ -143,7 +147,7 @@ To completely ignore the default CSS custom properties defined in the theme, add
 }
 ```
 
-## Section components
+### Section components
```

Comment:
These sections don't all neatly fit under "Customization" so I think we'd need to adjust the content a bit to change all the H2s to H3s

Reply:
Not a blocker BTW; noting this more to follow up on